pyomo/contrib/viewer/pyomo_viewer.py

```diff
@@ -12,9 +12,29 @@
 Run a Pyomo model viewer with embeded Jupyter QtConsole
 """
 
+import sys
 import logging
+import traceback
 _log = logging.getLogger(__name__)
 from pyomo.contrib.viewer.ui import *
+import pyomo.environ as pyo
```

Comment:
This import may cause issues if you intend any other auto-registered plugins to be importing this file.

Reply:
This file shouldn't be imported by anything else.  Its just a script to run the viewer. Anything you may want to import would be elsewhere.  To be safe though, I can move that back inside the main function.

Reply:
That's fine, then. Passing comment since I ran into something similar with one of my modules.